mods/ra/maps/survival01/survival01.lua

```diff
@@ -345,6 +368,7 @@ InitMission = function()
 	end)
 
 	Trigger.AfterDelay(DateTime.Seconds(1), function() Media.PlaySpeechNotification(allies, "MissionTimerInitialised") end)
+	TimerColor = HSLColor.New(115, 240, 130)
```

Comment:
`Player.Color` may be more convenient here. You just need to set it up with the Lua API in the `PlayerProperties.cs`.